src/main/java/org/launchcode/techjobs/oo/Location.java

```diff
@@ -2,53 +2,65 @@
 
 import java.util.Objects;
 
-public class Location {
-
-    private int id;
-    private static int nextId = 1;
-    private String value;
-
-    public Location() {
-        id = nextId;
-        nextId++;
-    }
-
-    // TODO: Add a constructor that takes a string as a parameter and assigns it to the 'value' field. The
-    //  constructor should also call the empty constructor in order to initialize the 'id' field.
-
-
-    // Custom toString, equals, and hashCode methods:
-
-    @Override
-    public String toString() {
-        return value;
+//public class Location {
+//
+//    private int id;
+//    private static int nextId = 1;
+//    private String value;
+//
+//    public Location() {
+//        id = nextId;
+//        nextId++;
+//    }
+//
+//    // TODO: Add a constructor that takes a string as a parameter and assigns it to the 'value' field. The
+//    //  constructor should also call the empty constructor in order to initialize the 'id' field.
+//
+//
+//    // Custom toString, equals, and hashCode methods:
+//
+//    @Override
+//    public String toString() {
+//        return value;
+//    }
+//
+//    @Override
+//    public boolean equals(Object o) {
+//        if (this == o) return true;
+//        if (!(o instanceof Location)) return false;
+//        Location location = (Location) o;
+//        return getId() == location.getId();
+//    }
+//
+//    @Override
+//    public int hashCode() {
+//        return Objects.hash(getId());
+//    }
+//
+//    // Getters and Setters:
+//
+//    public int getId() {
+//        return id;
+//    }
+//
+//    public String getValue() {
+//        return value;
+//    }
+//
+//    public void setValue(String value) {
+//        this.value = value;
+//    }
+//
+//}
+public class Location extends JobField{
+    public Location(String aLocation){
+        super(aLocation);
```

Comment:
```suggestion
    public Location(String value){
        super(value);
```
Great work on using `super()` to allow the subclass to access the constructor from the `JobField` class. To match the naming pattern used in the other job field types, you can change `aLocation` to value. It is good that you were specific with this name, though!